test/Table_Tests/src/Database/Postgres_Spec.enso

```diff
@@ -521,7 +521,9 @@ run_tests connection db_name =
 
     Common_Spec.spec prefix connection
 
-    common_selection = Common_Table_Operations.Main.Test_Selection.Config supports_case_sensitive_columns=True order_by_unicode_normalization_by_default=True allows_mixed_type_comparisons=False fixed_length_text_columns=True removes_trailing_whitespace_casting_from_char_to_varchar=True supports_decimal_type=True supported_replace_params=supported_replace_params
+    is_collation_en_gb_utf8 = connection.is_collation_en_gb_utf8
```

Comment:
IMO we should move the code checking for the collaction **_here_** instead of keeping it inside of the library.

Moreover, the code can be simplified, I'd suggest:
```
(connection.read "SELECT datcollate FROM pg_database WHERE datname = current_database()" . at "datcollate" . at 0) == "en_GB.UTF8"
```

Reply:
I wonder if this is not too narrow check though - wondering what settings are on our CI DB?

Maybe the check should be `(col != "C.UTF-8") && (col.contains "UTF8")` instead?

Reply:
> I wonder if this is not too narrow check though - wondering what settings are on our CI DB?
> 
> Maybe the check should be `(col != "C.UTF-8") && (col.contains "UTF8")` instead?

Ideally if you can, please check, to ensure that this test _does_ run on our CI. Otherwise it will be running only on my laptop and nowhere else 😅

I'd suggest adding an `IO.println` here and inspecting the logs of the CI workflow.